### DISCUSSION TITLE
qraphql호출 결과에서 병원 목록이 없어요.

### DISCUSSION THREAD

**Discussion by gitjung** [changdoc/naver-vaccine-macro#163] [Q&A]
개발자님 안녕하세요.제 가 메뉴얼 나온 대로 진행했는데 qraphql호출 결과에서 preview탭으로 이동할때 결과를 확장했는데 병원 목록이 하나도 없어요.어디서 잘 못 한 건가요?
<img width="848" alt="20210731_102758" src="https://user-images.githubusercontent.com/88194662/127724541-81e3e415-09f2-449f-8bb5-b62265ec21ad.png">
도와주세요~감사합나다

**Comment by changdoc** [comment]
business 쪽을 확장해보시면 목록을 발견 하실수 있을 거에요

**Reply by gitjung** [reply]
감사합니다.
